### DISCUSSION TITLE
mgos-to-tasmota failes with new_version ":"

### DISCUSSION THREAD

**Discussion by DSDV** [arendst/Tasmota#15819] [Q&A]
Hello, 

i bought two shelly plugs s and after getting them into my wifi and opening the link every thing looks like in the demo video except that there is no firmware flashing and the new_version shows ":"

i also tried using a different wifi and checked that the url is reachable. 

Is there a fix?

with kind regards

PS: after i tried this: https://github.com/arendst/Tasmota/discussions/15607#discussioncomment-2746053
they briked and dont work anymore .... same error tho (no diffrense when it comes to the message) the arendst links didnt brik my devices tho -.- 40 bucks down the drain 

sad to see 

//EDIT: 
Thats the output i got (both versions the same)
`{"status":"updating","has_update":false,"new_version":"","old_version":"20190516-073020/master@ea1b23db"}`

**Comment by barbudor** [comment]
You could have search before posting
Read https://github.com/arendst/Tasmota/discussions/15607

**Reply by DSDV** [reply]
i even linkted it in my post because it is what finally got me bricked - now i cant even return them. thanks to the search/recommendation in the link. 

arendst at least dindt brick the device. ...

**Reply by sfromis** [reply]
mgos-to-tasmota is not part of the Tasmota project, but https://github.com/yaourdt/mgos-to-tasmota
That may be a better place to check for others having issues or how to deal with it.

**Reply by DSDV** [reply]
yep and the 3rd time is the charm: its what bricked my plugs ...

geuss i can double down and try to flash them by serial - another 20 bucks down the drain ...

i m so frustrated -...

`this application should generally be safe to use for all supported devices`

**Reply by sfromis** [reply]
You "forgot" to quote the continuation of that....
> Still, overwriting a device's boot loader via OTA update is a risky operation. If something unexpected fails, your device may be bricked, unless you know how to flash a new firmware over a wired connection.

Anyway, the device is not too bad for wired flashing, meaning that the device should be quite recoverable. See
https://faulty.cloud/blog/shelly-plug-s-pinout
https://github.com/qtoggle/espqtoggle/wiki/Shelly-Plug-S

**Comment by Jason2866** [comment]
What is the intended message of this post? Is there a error in a described procedure where we (Tasmota team) can change "something" ?  We are not responsible nor related to the site https://github.com/yaourdt/mgos-to-tasmota
Theo forked and adopted this site for Tasmota useage.
Are following the procedures described here are ending in a soft bricked device?
If yes please describe what exactly have you done. Thanks!

**Comment by DSDV** [comment]
i got a usb uart device and was able to flash tasmota that way 

Also i noticed that after flashing tasmota only the red LED will work (when power is turned on). 
The blue indicator LED is completely dead after tasmota flash
